installation/scripts/installer.sh

```diff
@@ -95,6 +95,7 @@ kubectl apply -f - <<<"$COMBO_YAML"
 
 if [ $KNATIVE ]; then
     kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.knative": "true"}}'
+    kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.kymaEventBus": "false"}}'
```

Comment:
If those two publishers cannot be installed together in one cluster I would rely on one flag: `global.knative`. There should be only one flag to enable whole knative functionality. If we keep adding new flags it will quickly get unmaintainable. I'm not sure how to solve it in chart though. You probably need to wrap every resource in `{{ if not .global.knative }}`.

Reply:
Unfortunately, Helm doesn't support the `not` operator, that's why I had to have two :(

Reply:
Helm does not support not in `requirements.yaml`, but it does in charts. I cannot think of better way to do this, but in my opinion we cannot allow two flags to allow same thing.

Reply:
How about https://github.com/helm/helm/blob/master/docs/chart_template_guide/functions_and_pipelines.md#operators-are-functions

Reply:
Ok, let's leave it as is. Two points:
 * Please join those two flags in one patch
 * You also have to update [installation docs](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/04-09-knative.md)

Reply:
Thanks, @piotrmsc, and @sjanota for your review. I tried now different approaches to avoid the second flag but, unfortunately, none worked. I'll keep them in a single dict as advised.

Reply:
We talked offline about it and joing flags is fine